pywr/parameters/parameters.py

```diff
@@ -130,6 +130,27 @@ def load(cls, model, data):
 InterpolatedVolumeParameter.register()
 
 
+class InterpolatedLevelParameter(AbstractInterpolatedParameter):
```

Comment:
> It interpolates a value based on node flow at previous timestep

I think this should be named different. It implies it interpolates level not flow at the moment. Perhaps `InterpolatedFlowParameter`

Reply:
Yes, I agree with that